client/post-editor/editor-publish-date/post-scheduler.jsx

```diff
@@ -9,35 +9,28 @@ import { get } from 'lodash';
 /**
  * Internal dependencies
  */
-import AsyncLoad from 'components/async-load';
+import PostSchedule from 'components/post-schedule';
 import QueryPosts from 'components/data/query-posts';
 import postUtils from 'lib/posts/utils';
 import siteUtils from 'lib/site/utils';
 import { getSitePostsForQueryIgnoringPage } from 'state/posts/selectors';
 
-const PostSchedule = ( { onDateChange, onMonthChange, posts, selectedDay, site } ) => {
-	const tz = siteUtils.timezone( site );
-	const gmtOffset = siteUtils.gmtOffset( site );
-
+const PostScheduleWithCurrentMonthPosts = connect(
```

Comment:
This name doesn't really make sense, as the calendar shows posts for months other than the current month as well.

Reply:
"Current month" as in "the current month selected in the calendar".  In that sense, the name is accurate.

Do you have suggestions for an alternate name?  `s/Current/Selected/` maybe?

Reply:
Hmm... maybe something like `PostScheduleWithOtherPostsIndicated`?